src/pkg/packager/common.go

```diff
@@ -498,15 +530,16 @@ var (
 	ErrPkgSigButNoKey = errors.New("package is signed but no key was provided - add a key with the --key flag or use the --insecure flag and run the command again")
 )
 
-func (p *Packager) validatePackageSignature(publicKeyPath string) error {
+// ValidatePackageSignature validates the signature of a package
+func ValidatePackageSignature(directory string, publicKeyPath string) error {
 
-	// If the insecure flag was provided, or there is no aggregate checksum, ignore the signature validation
-	if config.CommonOptions.Insecure || p.cfg.Pkg.Metadata.AggregateChecksum == "" {
+	// If the insecure flag was provided, ignore the signature validation
+	if config.CommonOptions.Insecure {
 		return nil
 	}
```

Comment:
need a consensus on this

Reply:
We should not remove `|| p.cfg.Pkg.Metadata.AggregateChecksum == ""` as it will cause backwards compatibility for older packages to fail.

Reply:
I agree, but this is something we should mark for removal before GA

Reply:
I'm good with that - created an issue to track and add to here: https://github.com/defenseunicorns/zarf/issues/1956